fuseki/sample-ubd.trig

```diff
@@ -75,11 +75,17 @@ graph <cube-project/ubd/dataset> {
 <cube-project/ubd/dimensions-metadata> void:inDataset <ubd-example> .
 graph <cube-project/ubd/dimensions-metadata> {
   <cube-project/ubd/dimensions-metadata> a cc:DimensionMetadataCollection , hydra:Resource ;
+    schema:hasPart <cube-project/ubd/dimensions-metadata/dimension-value> ;
     schema:hasPart <cube-project/ubd/dimensions-metadata/dimension-year> ;
     schema:hasPart <cube-project/ubd/dimensions-metadata/station> ;
     schema:hasPart <cube-project/ubd/dimensions-metadata/aggregation> ;
     schema:hasPart <cube-project/ubd/dimensions-metadata/pollutant> ;
     schema:hasPart <cube-project/ubd/dimensions-metadata/unit> ;
+    schema:error
+      [
+        schema:identifier "MissingMeasureDimension" ;
+        schema:description "No Measure dimension defined" ;
+      ] ;
```

Comment:
The description to be displayed on the cube preview table
`identifier` is only used by the API so as not to create multiple same errors